CMakeLists.txt

```diff
@@ -309,6 +304,10 @@ if(NOT SFML_BUILD_FRAMEWORKS)
             DESTINATION .
             COMPONENT devel
             FILES_MATCHING PATTERN "*.hpp" PATTERN "*.inl")
+    install(DIRECTORY "${SFML_BINARY_DIR}/configured/"
```

Comment:
Can you use `PROJECT_BINARY_DIR` to better match existing style since `SFML_BINARY_DIR` is not currently used?

Reply:
Also, why not use one install call for both directories? That should be possible and minimizes redundant code.